content/1.2/docs/serverless-workflows/troubleshooting.md

```diff
@@ -94,3 +94,27 @@ a CA which is not available to the workflow. The error in the workflow pod log u
 
 **Solution:**
 When deploying a workflow in a namespace other than the one where Sonataflow services are running (e.g., sonataflow-infra), there are essential steps to follow to enable persistence and connectivity for the workflow. See the following [steps](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/release-1.2/README.md#additional-workflow-namespaces).
+
+### Problem: sonataflow-platform-data-index-service pods can't connect to the database on startup
+
+This can be cause by a network policy issue if you deployed RHDH Operator in a namespace different from `rhdh-operator`.
```

Comment:
Where is the connection between rhdh-operator and the DI pods not able to reach its DB?

We have NetworkPolicies that allows communication between sontaflow-infra et rhdh ns:
```
      - namespaceSelector:
          matchLabels:
            # Allow RHDH namespace to communicate with workflows and sonataflow services
            kubernetes.io/metadata.name: {{ .Values.rhdhOperator.subscription.targetNamespace }}
      - namespaceSelector:
          matchLabels:
            # Allow any other namespace the has workflows deployed because this is where
            # this namespace contains the sonataflow services
            rhdh.redhat.com/workflow-namespace: ""
```

If the user deployed the rhdh without our operator, this may indeed cause some issue for the communication between workflows' pods and RHDH when triggering execution